torch/onnx/utils.py

```diff
@@ -495,6 +495,19 @@ def unpack_quantized_tensor(value):
         return (value,)
 
 
+def _pre_trace_quant_model(model, args):
+    r"""Returns `torch.jit.trace(model, args)` if model is quantized. Otherwise do nothing and return
+    original model.
+
+    This is because the quantized weights are stored in special `PackedParams` structure. They need to
+    be handled by `torch.jit.trace` to be exportable to ONNX.
+    """
+    if (any([hasattr(m, "_packed_params") for m in getattr(model, "modules", lambda: [])()]) or
```

Comment:
I don't think you need a list inside `any`:
```suggestion
    if (any(hasattr(m, "_packed_params") for m in getattr(model, "modules", lambda: [])()) or
```